Release pipeline fails with cosign validation

https://github.com/k8gb-io/k8gb/actions/runs/7260158785/job/19779161323

```
  shell: /usr/bin/bash --noprofile --norc -e -o pipefail {0}
  env:
    DOCKER_CLI_EXPERIMENTAL: enabled
    COSIGN_PASSWORD: ***
    previous_tag: v0.11.5
INFO: Downloading bootstrap version 'v1.12.1' of cosign to verify version to be installed...
      https://storage.googleapis.com/cosign-releases/v1.12.1/cosign-linux-amd64
ERROR: Unable to validate cosign version: 'v1.12.1'
Error: Process completed with exit code 1.
```

--- a/.github/workflows/release.yaml
+++ b/.github/workflows/release.yaml
@@ -48,7 +48,7 @@ jobs:
           filterByMilestone: true
           unreleased: true
       - name: Install Cosign
-        uses: sigstore/cosign-installer@ced07f21fb1da67979f539bbc6304c16c0677e76 # renovate: tag=v2.7.0
+        uses: sigstore/cosign-installer@9614fae9e5c5eddabb09f90a270fcb487c9f7149 # renovate: tag=v3.3.0
         with:
           cosign-release: 'v1.12.1'
       - name: Install Syft

